addons/l10n_ar_website_sale/controllers/main.py

```diff
@@ -0,0 +1,57 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import _
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+from odoo.http import request, route
```

Comment:
@jco-odoo 
This change should solve it. Could you make a commit and squash? or maybe I can do a new pr for this FW..

```suggestion
from odoo.http import request
```